Specs/Widgets/InfoBox/InfoBoxViewModelSpec.js

```diff
@@ -14,6 +16,13 @@ defineSuite([
         expect(viewModel.maxHeightOffset(0)).toBeDefined();
     });
 
+    it('sets title', function() {
```

Comment:
What is the goal of this test?  It passes in master without any of your changes because `titleText` is the same no matter what, it's how it's parsed that changes in this branch (if security is disabled).